packages/pds/tests/_util.ts

```diff
@@ -159,16 +160,6 @@ export const runTestServer = async (
   }
 }
 
-const usedLockIds = new Set()
-const uniqueLockId = () => {
```

Comment:
snuck this in - I wonder if it was the source of some of our pds test failures on leader - different tests could get the same lock id because we were tracking two separate sets

Reply:
Haha, oh lord!